sdk/lusid/utilities/api_client_builder.py

```diff
@@ -52,6 +52,8 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         # Load the configuration
         configuration = ApiConfigurationLoader.load(api_secrets_filename)
 
+
```

Comment:
Please remove this empty space see - https://www.python.org/dev/peps/pep-0008/#blank-lines